Tools/BaseBuild/README.md

```diff
@@ -89,6 +89,25 @@ oc process -f ./adminWebClientSecrets.yaml -p OIDC_SECRET=[Client OIDC Secret]
 oc set env --from=secret/adminwebclient-secrets dc/adminwebclient
 ```
 
+### WebClient Production Only Robots.txt Configuration
+
+Health Gateway has a configurable [robots.txt](../../Apps/WebClient/src/Server/Controllers/RobotsController.cs) that is based on the HealthGateway_Robots.txdt environment variable.  Our default is to have all Robots disallowed in non-Production environments.
```

Comment:
is it "..._Robots.txt" or "..._Robot.txdt"

Reply:
Probably a nit, but I've resolved.

Reply:
haha I always have typo in my writing, so I think it helps me to spot these :-)